mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -257,6 +289,13 @@
         <Flags index="AutoCast" value="1"/>
         <Flags index="AutoCastOn" value="1"/>
     </CAbilEffectTarget>
+    <CAbilBuild id="CreepTumorBuild">
+        <InfoArray index="Build1">
+            <Cooldown TimeStart="18" TimeUse="18"/>
```

Comment:
Creep Tumor: Cooldown increased from 11 to 13 seconds (from 15 to 18 on normal speed)

The build time is still 11 seconds (15 on normal speed)